src/components/charts/bar-chart.js

```diff
@@ -43,68 +43,61 @@ const BarChart = ({
   )
 
   return (
-    <div align={align}>
```

Comment:
I had not idea that `align` was even a valid attribute for `<div>` elements! Did a bit of research and it looks like [MDN/caniuse](https://caniuse.com/#feat=mdn-html_elements_div_align) says that this attribute is deprecated and shouldn't be used. Noting for posterity

Reply:
Yeah, I didn't think we should use it, and the layout is now handled by classes with flexbox.